app/models/mixins/supports_feature_mixin.rb

```diff
@@ -69,6 +69,7 @@ module SupportsFeatureMixin
     :resize                   => 'Resizing',
     :retire                   => 'Retirement',
     :smartstate_analysis      => 'Smartstate Analaysis',
+    :ems_network_new          => 'New EMS Network Provider'
```

Comment:
@Ladas : we added this so we could use support mixin for  creating  nuage as a network provider